fava/templates/source.html

```diff
@@ -15,7 +15,7 @@
                 </tr>
             </thead>
             <tbody>
-                {% for source_file in api.source_files() %}
+                {% for source_file in api.source_files()|sort %}
```

Comment:
Ah OK I see. Should be sorted here, then:

https://github.com/aumayr/fava/blob/master/fava/api/__init__.py#L455